modules/ROOT/pages/platform/security.adoc

```diff
@@ -255,3 +255,79 @@ TLS v1.2:
 * `TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (RFC5289)`
 * `TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256 (RFC7905)`
 * `TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 (RFC5288)`
+
+== Customer managed keys
```

Comment:
```suggestion
== Customer-managed keys
```

I _think_ this should have a hyphen in it. See for example:

- https://cloud.google.com/kms/docs/cmek
- https://learn.microsoft.com/en-us/azure/security/fundamentals/key-management
- https://www.databricks.com/trust/security-features/data-protection-with-customer-managed-keys

Reply:
Went with AWS's "customer managed keys" spelling, and looked better to me without a hyphen